pages/index.js

```diff
@@ -37,48 +33,31 @@ const StyledButton = styled.button`
   width: 3rem;
 `;
 
-const StyledList = styled.ul`
-  list-style: none;
-  padding: 0;
-  margin: 2rem auto;
-  text-align: left;
-`;
-
-const StyledListItem = styled.li`
-  margin: 1.2rem auto;
-`;
-
 const StyledLink = styled(Link)`
   text-decoration: none;
-  padding: 1rem;
-  background-color: lightgray;
-  border: 1px solid black;
-  border-radius: 5px;
   color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  text-align: center;
+  &:hover {
+    background-color: lightskyblue;
+  }
```

Comment:
As mentioned above. The `:hover` style will not have any effect on touch devices. But this is more a refactor thing. ✨